core/templates/dev/head/components/button-directives/hint-and-solution-buttons.directive.html

```diff
@@ -73,6 +73,9 @@
     margin-top: 5px;
     padding: inherit;
     width: 32px;
+    min-width: 22px;
```

Comment:
Please keep this alphabetized, see our [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

Reply:
Updated Thanks!